app/_styles/scss/main.module.scss

```diff
@@ -0,0 +1,74 @@
+.html {
+    height: 100%;
+    margin: 0;
+}
+
+.mainStyle {
+    display: flex;
+    flex-direction: column;
+    min-height: 100vh;
+}
+
+.content {
+    flex: 1;
+}
+
+// error
+
+.wrapperError {
```

Comment:
dublication 
please remove from main.module.scss and keep in errors.module.scss